templates/geosafe/analysis/dynamic_scripts/create/main.html

```diff
@@ -215,7 +215,7 @@
                 $("#impact-card-modal").replaceWith(data);
                 $("#impact-card-modal").modal();
                 var impact_layer_title = $("#impact-option a[data-id='" + selected_impact_id + "']").text();
-                show_info_text('Show Analysis Summary', 'Showing summary for ' + impact_layer_title);
+                show_info_text('{% trans 'Show Analysis Summary' %}', '{% trans 'Showing summary for ' %}' + impact_layer_title);
```

Comment:
Can you try using template interpolation @myarjunar ?
Refer to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

You could possibly write something like this:
```suggestion
                show_info_text('{% trans 'Show Analysis Summary' %}', '{% trans 'Showing summary for ${impact_layer_title}' %}');
```
So that translator can reorder the title with better context.